README.md

```diff
@@ -1 +1,63 @@
-# Shuttle-UI
\ No newline at end of file
+
+<img src="./docs-site/assets/logo.png" width="150" height="150">
```

Comment:
Please use alt attribute here.

Reply:
Thanks for pointing it out. I have fixed it.